yagb-core/src/emulator/ppu/ppu-cgb.ts

```diff
@@ -102,6 +112,9 @@ export class PpuCgb extends PpuBase {
 
         this.frontBuffer.fill(PALETTE_CLASSIC[4]);
         this.backBuffer.fill(PALETTE_CLASSIC[4]);
+
+        this.bcram.fill(0xff);
+        this.ocram.fill(0);
```

Comment:
Do you have a reason for the difference between `bcram` and `ocram`? Otherwise, just initialise them both to 0xff for now.

Reply:
From pandoc:
All background colors are initialized as white by the boot ROM, however it is a good idea to initialize all colors yourself, e.g. if implementing a soft-reset mechanic.
The boot ROM leaves all object colors uninitialized (and thus somewhat random), aside from setting the first byte of OBJ0 color #0 to $00, which is unused.

Reply:
Ah, that's what I was looking for. 👍